app/components/a_level_row_component.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= govuk_inset_text(classes: inset_text_css_classes) do %>
+  <p class="govuk-heading-s app-inset-text__title">A-levels and equivalency tests</p>
+  <p class="govuk-body">
+    <%= govuk_link_to "Enter A-levels and equivalency test requirements",
```

Comment:
We have been using A levels, not A-levels in the designs and elsewhere in the code base. 

I wonder too, can we add the text to the translation file instead of hard coding it here?

Reply:
Done